### ISSUE TITLE
Позиционирование блока ссылок на внешние карты и поведение кнопки

### ISSUE THREAD

**Issue by myadzel** [wladich/nakarte#437]
Предложу сделать кнопку схожей по поведению с панорамами, дистанцией и координатами, а именно так, чтобы кнопка открывала и закрывала блок с ссылками. 

Сейчас по клику нельзя закрыть блок, нужно выполнить какое-либо действие. Это немного напрягает.

**Comment by wladich** [comment]
Это не блок, а обычное меню, его можно закрыть кликом по карте или нажатием Esc.
Делать постояяно видимый блок, мне кажется, неправильно, т.к. это именно одноразовое действие, т.е. один раз перешел и больше инструмент не нужен.

**Comment by myadzel** [comment]
Тебе виднее, но мне кажется, что было бы очевиднее. 

Вот еще из подобного, если уж затронули кнопки слева. Инструмент измерения дистанции и азимута делает то, что как обозначена кнопка "Measure distance", хотя та по сути не измеряет, а добавляет трек с именем "Ruler".

**Comment by wladich** [comment]
Это да, кнопка Ruler это историческое наследие, предполагалось для лучшей понятности для привыкших к Яндекс картам. С тех пор и Яндекс совсем поменялся и непонятный Азимут появился.

**Comment by wladich** [comment]
Готов обсудить переименования, но кнопку Ruler не уберу, я ею постоянно пользуюсь.

**Comment by wladich** [comment]
А вот в полезности Азимута у меня большие сомнения...

**Comment by myadzel** [comment]
> А вот в полезности Азимута у меня большие сомнения...

Прикрутить Яндекс.Метрику и задать цели.

**Comment by myadzel** [comment]
> Готов обсудить переименования, но кнопку Ruler не уберу, я ею постоянно пользуюсь.

Но ведь она делает то же самое, что и кнопка "New track" (кстати, хорошо бы здесь new заменить на глагол действия --- add/create), только километры сразу включены у ней.

**Comment by wladich** [comment]
Да, именно так и было задумано - на Яндексе в этом же месте была кнопка с линейкой - начать измерение по ломанной. Предполагается, что человек, не знакомый с интерфейсом, если захочет померить расстояние, то нажмёт на линейку (и подсказка Measure distance ему в этом поможет). А то, что у трека можно включить отметки километров это гораздо сложнее показать-объяснить.

**Comment by wladich** [comment]
> Прикрутить Яндекс.Метрику и задать цели.

Да у меня и своя телеметрия есть, просто мерить надо с какой-то целью. Цель тут одна - убрать, а я пока её совсем убирать не хочу.
